src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/ChangeProviderInform.cshtml

```diff
@@ -14,13 +15,15 @@
     </h1>
     <div class="govuk-!-width-two-thirds das-panel__body">
 
-        @if (Model.ApprenticeshipStatus != ApprenticeshipStatus.Stopped)
+        @if (Model.ApprenticeshipStatus != ApprenticeshipStatus.WaitingToStart ||
+            Model.ApprenticeshipStatus != ApprenticeshipStatus.Live ||
+            Model.ApprenticeshipStatus != ApprenticeshipStatus.Paused)
```

Comment:
This will always be true even if status is stopped, it should be == || ==

Reply:
good spot